src/test/java/core/basesyntax/dao/FruitDaoImplTest.java

```diff
@@ -2,7 +2,7 @@
 
 import static org.junit.Assert.assertEquals;
 
-import core.basesyntax.db.FruitsStorage;
+import core.basesyntax.dbfortest.FruitStorageForTests;
```

Comment:
i was not suggesting to make another class for tests, just to make a map parameter for dao and to put there Storage or some dummy map, but your way can be fine.